### PR TITLE
Update example notebooks

### PR DESCRIPTION
**Pull request by tomicapretto**
I updated some notebooks to reflect changes in Bambi and some extra comments/explanations I added when considered they could be helpful. I include the WIP label because I think the notebooks may be modified before merging according to feedback and discussion.

**Note:** In a chat with @aloctavodia  we talked about modifying the object that is passed to `az.plot_trace()` in such a way that `*_offset` terms are not included by default. This is not included in this PR. I think we should work on it in a separate PR and then update the examples again to reflect changes.

### CODE REVIEW

File: docs/notebooks/bayesian_t-test.ipynb [hunk 1]
```diff
@@ -38,14 +38,17 @@
    "cell_type": "markdown",
```
Comment (aloctavodia) [inline docs/notebooks/bayesian_t-test.ipynb:38]: <p>Bambi</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bambinos/bambi/pull/232/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Comment (aloctavodia) [inline docs/notebooks/bayesian_t-test.ipynb:38]: <p>use singular for posterior?</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bambinos/bambi/pull/232/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Comment (aloctavodia) [inline docs/notebooks/bayesian_t-test.ipynb:38]: <p>add sentence about how much different are the means, not just that they are different</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bambinos/bambi/pull/232/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Comment (tomicapretto) [reply docs/notebooks/bayesian_t-test.ipynb:38]: <p>I'm sorry I don't understand. What do you mean with singular?</p>

Comment (aloctavodia) [reply docs/notebooks/bayesian_t-test.ipynb:38]: <p>posterior vs posterior<strong>s</strong></p>

**Pull request merged**